snippet-extractor-output/snippets.yaml

```diff
@@ -2500,6 +2500,139 @@
 
         await context.sync();
     });
+'Excel.FormattedNumberCellValue#basicType:member':
```

Comment:
Since this sample is repeated on the ref doc page, consider omitting one for `basicType`. That property is just read-only and not as interesting from a scenario perspective.

Reply:
Good point!